int32 TaskID
int32 TaskListID
int32 TaskType
int32 LocationType
float32 CoordinatesX
float32 CoordinatesY
float32 CoordinatesTetha
bool IsRunning
bool IsDone
bool IsCanceled